src/external/song.ts

```diff
@@ -17,100 +17,186 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import abortController from 'abort-controller';
+import { isArray } from 'lodash';
 import fetch from 'node-fetch';
-import Batch from 'stream-json/utils/Batch';
+import qs from 'qs';
```

Comment:
had to look up what this package does... default imports can be easily named in a declarative manner.

```suggestion
import queryStrings from 'qs';
```